lib/tasks/send_owner_emails.rake

```diff
@@ -18,4 +18,25 @@ namespace :fromthepage do
     end
   end
 
-end
\ No newline at end of file
+  desc "monthly owner email wrap ups"
+  task :monthly_owner_wrapup => :environment do
+    owners = User.where(owner: true).where.not(account_type: [nil, 'Trial'])
+    if SMTP_ENABLED
+      puts "Sending Monthly Wrapup Email to:"
```

Comment:
This `SMTP_ENABLED` conditional was inside this method when I started doing work on this branch. I see the `SMTP_ENABLED` constant in several places in the codebase, so I am not sure what all it is used for and didn't want to remove it. However, with `mailcatcher` set up in the `environments/development.rb` file, I am not sure it is needed.